rules/use-prefixes-to-improve-code-review-communication/rule.md

```diff
@@ -30,12 +30,11 @@ Figure: Without a prefix, this comment's intent is vague. It's not evident wheth
 :::
 
 :::greybox
-@bob-northwind
+Hi @bob-northwind
```

Comment:
```suggestion
@bob-northwind
```
This is an example Code Review comment from Bob Northwind, hence the clarification in the previous comment.